authors/apps/comments/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+
+from authors.apps.articles.models import Article
+from authors.apps.profiles.serializers import ProfileSerializer
+
+from .models import Comment
+
```

Comment:
Please kindly look into the lines required by pep8 for empty lines

Reply:
> 
> 
> Please kindly look into the lines required by pep8 for empty lines

Kindly elaborate more on this. Which empty lines are you refering to as I already run this file through autopep8.